core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -237,6 +237,7 @@ public void trackMetric(String name, double value, int sampleCount, double min,
         if (sampleCount > 1) {
             mt.setMin(min);
             mt.setMax(max);
+            mt.setStandardDeviation(0.0);
```

Comment:
As I commented in the PR: This change should happen in the MetricTelemetry class. Basically there is no need of the following logic : 

```if ((metric.getCount() != null) && metric.getCount() == 1) {
            // Singular data point. This is not an aggregation.
            isAggregation = false;
        }
```
as aggregation is always true irrespective of count and this is how backend treats it. If you would like you can confirm with Greg, but I am pretty sure about this based on my ongoing work on metrics' conversation with him.